test/sampleApp/index.html

```diff
@@ -474,7 +474,15 @@ <h3>Wrapped in a <code>&lt;figure&gt;</code> element, with a <code>&lt;figcaptio
                     <legend>Action buttons</legend>
                     <p>
                         <input type="submit" value="<input type=submit>">
-                        <input type="button" value="<input type=button>">
+                        <input id="change-name-btn" type="button" value="Close Curtain" onclick="changeValue()">
```

Comment:
what this change for ?

Reply:
It was don't to support the get text tests hover at the end we used something else.
Change the script to be changeValue and it can be used in future tests to check value change.